compiler/token_name.cpp

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright: Copyright nsoul97 (Soulounias Nikolaos) 2019.
+ * License:   $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Authors:   nsoul97 (Soulounias Nikolaos)
+ */
+
+#include "token_name.h"
+#include "tokens.h"
+#include <cassert>
+
+/*
+The function returns the name of the token that corresponds to the enumeration
+which is passed as argument.
+*/
+const char *token_name(TOK kind) {
```

Comment:
Please remove the `token_name*` files because they implement another issue :p

Reply:
Should I add them with another pr? I checked that they are not assigned to anyone.

Reply:
Yep.